conf/production.ini

```diff
@@ -0,0 +1,42 @@
+[pipeline:main]
```

Comment:
Bouncer didn't have a `production.ini`, I just copy-pasted the one from test-pyramid-app